packages/lesswrong/components/form-components/ImageUpload.tsx

```diff
@@ -29,28 +30,31 @@ const styles = (theme: ThemeType): JssStyles => ({
   imageIcon: {
     marginRight: theme.spacing.unit
   },
+  removeButton: {
+    marginLeft: 10
+  }
 });
 
 const cloudinaryArgsByImageType = {
   gridImageId: {
-    min_image_height: 80,
-    min_image_width: 203,
-    cropping_aspect_ratio: 2.5375,
-    upload_preset: cloudinaryUploadPresetGridImageSetting.get(),
+    minImageHeight: 80,
+    minImageWidth: 203,
+    croppingAspectRatio: 2.5375,
+    uploadPreset: cloudinaryUploadPresetGridImageSetting.get(),
```

Comment:
The new widget [uses camel case](https://cloudinary.com/documentation/upload_widget_reference#parameters)